.github/workflows/playground_selenium.yaml

```diff
@@ -2,38 +2,77 @@ name: Playground Selenium Tests
 
 on:
   workflow_dispatch:
+    inputs:
+      branch:
+        description: "Select which Git branch to run against"
+        required: true
+        default: "development"
+      network:
+        description: "Select the target environment"
+        required: true
+        type: choice
+        default: "dev"
+        options:
+          - local
```

Comment:
The `local` option will run the dashboard server on which network?

Reply:
will do a local lerna build and run locally;
and this was the originally daily night run.
but i think we don't need it anymore and better to run nightly run on staging.
what do you think ?

Reply:
ok, but it's not a network cause the local server could run against dev, qa, test, and main nets. Also, the branch option will only run the tests from this branch, but the source code will be based on the deployed website. So IMO, we should always run the server locally with the preferred network.

here is the guide to switch the network https://github.com/threefoldtech/tfgrid-sdk-ts/blob/development/packages/playground/docs/build.md#run-the-script

Reply:
what we can do is 
keep the default nightly run install the server locally to have the same branch.

but for other networks don't see the point of that;
we can use the tests from any branch to run against any network release,
this will help us more in release actually.

but will keep your point only for nightly run on local build.